### ISSUE TITLE
Creating zpool on windows 10 pro

### ISSUE THREAD

**Issue by cyberax64** [openzfsonwindows/ZFSin#250]
good evening,

I tested the ZFS again on windows and I came across an error:

C:\WINDOWS\system32>wmic diskdrive list brief
Caption                                DeviceID            Model                                  Partitions  Size
WDC WD40EZRX-00SPEB0                   \\.\PHYSICALDRIVE3  WDC WD40????-00SPEB0                   0           4000784417280
WDC WD40EZRZ-00GXCB0                   \\.\PHYSICALDRIVE4  WDC WD40????-00GXCB0                   0           4000784417280
WDC WD10JPVX-80JC3T0                   \\.\PHYSICALDRIVE1  WDC WD10????-80JC3T0                   1           1000202273280

C:\WINDOWS\system32>zpool create tank PHYSICALDRIVE3
Expanded path to '\\?\PHYSICALDRIVE3'
DeviceName said OK with '*'
DriveGeometry said OK
LayoutInfo said OK
DiskExtents said NG
warning: device in use checking failed: Invalid argument

**Comment by cyberax64** [comment]
OK
after checking the hard drive is not initialized (and you better saved 4TB of data that I would have lost : o)

**Comment by lundman** [comment]
Have you tried creating a file-based pool to see if that at least works?

**Comment by cyberax64** [comment]
yes i tested and it works, i also reinstall my pc on a hard disk by activating the UEFI function and its works too :)
the problem comes from the Legacy system

**Comment by jackybj007** [comment]
it also failed in windows10 under Legacy system. When I run PowerShell as Administrator rights, it works well. so it may be a authority issue.

**Comment by jackybj007** [comment]
> yes i tested and it works, i also reinstall my pc on a hard disk by activating the UEFI function and its works too :)
> the problem comes from the Legacy system

try run PowerShell as Administrator rights, it will works well

**Comment by jackybj007** [comment]
> yes i tested and it works, i also reinstall my pc on a hard disk by activating the UEFI function and its works too :)
> the problem comes from the Legacy system

try run PowerShell as Administrator rights, it will works well

**Comment by vmlemon** [comment]
I found that deleting all of the partitions, on the drive, converting it from GPT, to MBR labellling, and then rebooting, would let me create a ZPool on an old SSD, with Windows 10.0.18363.1316, FWIW, after encountering this error, on a new machine, recently.

**Comment by fcmircea** [comment]
I have the same issue, have tried the disk both as GPT and MBR.

I use:

`zpool create -f -O casesensitivity=insensitive -O compression=lz4 -O atime=off -o ashift=12 -fnd win_big \\.\PHYSICALDRIVE2`

The disk is empty, trying to `-f` to force somehow.

Any help appreciated.
